appcontainer/Dockerfile

```diff
@@ -1,51 +1,20 @@
-FROM python:3.10
-
-ENV PYTHONDONTWRITEBYTECODE=1 \
-    PYTHONUNBUFFERED=1 \
-    USER=calitp
-
-    # create non-root $USER and home directory
-RUN useradd --create-home --shell /bin/bash $USER && \
-    # setup $USER permissions for nginx
-    mkdir -p /var/cache/nginx && \
-    chown -R $USER /var/cache/nginx && \
-    mkdir -p /var/lib/nginx && \
-    chown -R $USER /var/lib/nginx && \
-    mkdir -p /var/log/nginx && \
-    chown -R $USER /var/log/nginx && \
-    touch /var/log/nginx/error.log && \
-    chown $USER /var/log/nginx/error.log && \
-    touch /var/run/nginx.pid && \
-    chown -R $USER /var/run/nginx.pid && \
-    # setup directories and permissions for Django and gunicorn
-    mkdir -p /home/$USER/app/config && \
-    mkdir -p /home/$USER/app/run && \
-    mkdir -p /home/$USER/app/static && \
```

Comment:
Looks like we lost this creation of the `static` directory - it's not in the base. Not sure if it is entirely needed?

Reply:
From my testing where I built from scratch, everything seems to still work even without creating `static`. 🤷

Reply:
Yeah I guess even the output from the port issue we were discussing earlier showed that static files were being copied.

Let's just make sure they can be _read_ too when running the app container (i.e. not debug/devcontainer)

Reply:
I ran `docker compose up client` and when I launched the app in my browser, the CSS and images all loaded ok. Are there any other static files I should check for?

Reply:
Sounds good!